src/background.ts

```diff
@@ -52,6 +53,22 @@ class ShortExt {
             });
     }
 
+    isEmptyTab(tab: chrome.tabs.Tab) {
+        return !tab.hasOwnProperty("url");
+    }
+
+    onIconClick = (webUi: string) => {
+        chrome.browserAction.onClicked.addListener(tab => {
+            if(this.isEmptyTab(tab)) {
+               return;
+            }
+            if(tab.url!.includes("https://s.time4hacks.com")) {
+                return;
+            }
```

Comment:
May I ask what is this for? Also let's use webUi instead of hard coding URL here